smtp.go

```diff
@@ -101,6 +102,15 @@ type DeliverByOptions struct {
 	Trace bool
 }
 
+type PriorityProfile string
+
+const (
+	NONE       PriorityProfile = ""
+	MIXER      PriorityProfile = "MIXER"
+	STANAG4406 PriorityProfile = "STANAG4406"
+	NSEP       PriorityProfile = "NSEP"
```

Comment:
Naming:

- Can we prefix the priority profile names with `Priority`? `NONE` alone is too generic.
- Idiomatic Go names don't use UPPERCASE, they use camelCase.